content/project/podcast/index.md

```diff
@@ -0,0 +1,42 @@
+---
+date: 2022-08-11
+title: "GFSC Podcast"
+client: "GFSC Collective"
+when: "2022 - now"
+who: kim rachele mallory zara david honor
+themes: digital-autonomy anti-racism trans-lib hulme-and-manchester mutual-aid
+categories: publications
+image: index.jpg
+imagealt: The Geeks for Social Change Podcast
+social: social.jpg
+description: "A podcast about navigating technology, ethics, and activism in our day-to-day lives and work, created with our friends, collaborators and co-conspirators."
+weight: 100
+galleryalt:
+  - A graphic reading 'Introducing... The Geeks for Social Change Podcast' with faces of podcast contributors beneath
+  - A graphic reading 'How government technology limits who we can become' with faces of contributors beneath it
+  - The rise and fall of Facebook Events with contributors photos
+  - Everything we want to tell you about your funding scheme but are afraid to tell you with contributors photos
+ctaButtons:
+  - text: RSS 
+    url: https://anchor.fm/s/a948e35c/podcast/rss
+  - text: Spotify
+    url: https://open.spotify.com/show/5W8FIdNNr3i1jPj02uUevO
+  - text: Radio Public 
+    url: https://radiopublic.com/geeks-for-social-change-60Xbpb
+  - text: Apple Podcasts 
+    url: https://podcasts.apple.com/us/podcast/geeks-for-social-change/id1639344512
+  - text: Amazon Music
+    url: https://music.amazon.co.uk/podcasts/e37fd22d-7c11-4cc9-a282-951cc205733c/geeks-for-social-change
+  - text: Spotify for Podcasters
+    url: https://anchor.fm/gfscstudio/
+draft: false
+---
+Our podcast aims to explore the many themes and topics that are of interest to us as a studio, via a series of conversations with our friends, collaborators and co-conspirators. We focus on navigating technology, ethics, and activism in our day to day lives and work.
+
+We also include some special episodes — members of the studio occasionally deliver talks or panels at external events, and where possible, we aim to record these for our podcast so that they can be enjoyed by a much wider audience. 
+
+In a 2022 survey we discovered that over 60% of our extended community regularly listen to podcasts. Many members of the studio also find podcasts a vital and enriching medium for knowledge sharing and entertainment. 
+
+Our podcast does not currently operate to a fixed schedule, as this enables us to ensure the highest possible quality and relevance for each episode (basically, we make them when we feel like it!) — be sure to subscribe via your podcasting service of choice, to make sure you never miss an episode.
```

Comment:
Would be great to link to the blogs containing the transcripts from in here or maybe just signpost that they are visible in the linked blog posts displayed below?

Reply:
signposting!